src/test/java/com/fasterxml/jackson/databind/ser/CanonicalBigDecimalToString.java

```diff
@@ -0,0 +1,43 @@
+package com.fasterxml.jackson.databind.ser;
```

Comment:
These should probably be put under new package (not sure if `databind.canonical` or `databind.ser.canonical`).

I realize all code is under `src/test` and not necessarily meant to be in specific location.